ratdb/AMBE_NSPECTRUM.ratdb

```diff
@@ -0,0 +1,28 @@
+// Neutron energy spectrum of a AmBe source simulated with MCNP (NIM A Volume
+// 763, 1 November 2014, Pages 547-552)
+{
+	name: "AMBE_NSPECTRUM",
+	index: "ambe_n",
```

Comment:
I'm not sure you need the index here, since it doesn't add any information. You can just leave it as an empty string.